src/main/java/frc/robot/subsystems/intake/IntakeSubsystem.java

```diff
@@ -375,19 +375,33 @@ private boolean checkSimConveyor(double expectedSpeed) {
 
   @Override
   public void periodic() {
-    // Intake Rollers
-    Logger.recordOutput("Intake/Rollers/Motor Output", intakeMotor.get());
-    Logger.recordOutput(
-        "Intake/Rollers/Motor Voltage (Volts)", intakeMotor.getMotorVoltage().getValue().in(Volts));
+    if (!Robot.isCompetition) {
 
-    // Convayor
-    Logger.recordOutput("Intake/Conveyor/Motor Output", conveyorMotor.get());
-    Logger.recordOutput(
-        "Intake/Conveyor/Motor Voltage (Volts)",
-        conveyorMotor.getMotorVoltage().getValue().in(Volts));
-    Logger.recordOutput(
-        "Intake/Conveyor/Velocity (RPS)",
-        conveyorMotor.getVelocity().getValue().in(RotationsPerSecond));
+      // Intake Rollers
+      Logger.recordOutput("Intake/Rollers/Motor Output", intakeMotor.get());
+      Logger.recordOutput(
+          "Intake/Rollers/Motor Voltage (Volts)",
+          intakeMotor.getMotorVoltage().getValue().in(Volts));
+
+      // Conveyor
+      Logger.recordOutput("Intake/Conveyor/Motor Output", conveyorMotor.get());
+      Logger.recordOutput(
+          "Intake/Conveyor/Motor Voltage (Volts)",
+          conveyorMotor.getMotorVoltage().getValue().in(Volts));
+      Logger.recordOutput(
+          "Intake/Conveyor/Velocity (RPS)",
+          conveyorMotor.getVelocity().getValue().in(RotationsPerSecond));
+
+      // Pose 3D of Intake
+      Angle currentAngle = getPivotAngle().times(-1);
+      Angle constantAngle = DrivetrainConstants.kIntakeStartPose.getRotation().getMeasureY();
+      Angle combinedAngle = currentAngle.plus(constantAngle);
+      Logger.recordOutput(
+          "Odometry/Mech Poses/Intake Pose",
+          new Pose3d(
+              DrivetrainConstants.kIntakeStartPose.getTranslation(),
+              new Rotation3d(0, combinedAngle.in(Radians), 0)));
```

Comment:
We should keep this logging all the time, its a great thing to show the judges, and can kinda be useful.